supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -323,13 +323,13 @@ There is no functional difference between the first server that was installed an
 *See also*:
 
 [[ingress]]
-==== image:images/yes.png[yes] Ingress (noun)
-*Description*: In Red Hat OpenShift, _Ingress_  is an API object developers can use to expose services through an HTTP(S) aware load balancing and proxy layer through a public DNS entry. The Ingress resource might further specify TLS options and a certificate, or specify a public CNAME that the OpenShift Ingress Controller should also accept for HTTP and HTTPS traffic. An administrator typically configures their Ingress Controller to be visible outside the cluster firewall and might also add additional security, caching, or traffic controls on the service content.
+==== image:images/yes.png[yes] ingress (noun) (adjective)
+*Description*: In Red Hat OpenShift, _Ingress_  is an API object that developers can use to expose services within the cluster through an HTTP(S) aware load balancer and a proxy layer using a public DNS entry. The `Ingress` resource defines the cluster-wide configuration for ingress traffic, and provides the ability to specify TLS options, a certificate, or a public CNAME that the OpenShift Ingress Controller (`IngressController`) can accept for HTTP(S) traffic. Additionally, _ingress_ can also be used to describe the incoming direction of network traffic. In Red Hat OpenShift, for example, this traffic is described as entering (ingress) or leaving (egress) an OCP cluster.
```

Comment:
```suggestion
*Description*: In Red Hat OpenShift, _Ingress_  is an API object that developers can use to expose services within the cluster through an HTTP(S) aware load balancer and a proxy layer using a public DNS entry. The `Ingress` resource defines the cluster-wide configuration for ingress traffic, and provides the ability to specify TLS options, a certificate, or a public CNAME that the OpenShift Ingress Controller (`IngressController`) can accept for HTTP(S) traffic. Additionally, _ingress_ can also be used to describe the incoming direction of network traffic. In Red Hat OpenShift, for example, this traffic is described as entering (ingress) or leaving (egress) an OCP cluster.
```

1. Suggest putting each sentence on a new line to make it easier for reviewers to add comments :)
2. Is the first sentence necessary? We should avoid using "Ingress" on its own. 
3. What noun should be used after `IngressController`? Is it a resource or an object or something else?
4. "OCP" is not an approved abbreviation, do not use.
5. Should we add an entry for "egress", for completeness?

Reply:
1. ACK, good to know. Since this has already gone through most of the reviews its going to have, I think I'll forego and just make sure to do this for future style guide contributions :) Thanks for the tip though!
2. The first sentence is necessary, and I think it should be clarified to say that "_ingress_ is a Kubernetes API object". I was just following the convention of italicizing the term being defined, although you are correct that technically this usage of Ingress object would typically get backticks. 
3. resource. added!
4. ACK. Changed to OpenShift. 
5. That's a good idea. I would prefer to handle that addition in a separate PR, just so I can go ahead and merge this one. I opened it in July 😓